api/src/test/java/com/epam/pipeline/test/creator/pipeline/PipelineCreatorUtils.java

```diff
@@ -30,9 +42,18 @@ private PipelineCreatorUtils() {
     }
 
     public static Pipeline getPipeline(final String owner) {
-        Pipeline pipeline = new Pipeline();
+        final Pipeline pipeline = new Pipeline();
         pipeline.setId(ID);
         pipeline.setOwner(owner);
+        pipeline.setCurrentVersion(getRevision());
+        return pipeline;
+    }
+
+    public static Pipeline getPipeline(final Long id, final String owner, final Long parentId) {
+        final Pipeline pipeline = new Pipeline();
+        pipeline.setId(id);
+        pipeline.setOwner(owner);
+        pipeline.setParentFolderId(parentId);
         return pipeline;
     }
```

Comment:
It's a good practice to call overloaded methods from each other. 

In this case it is possible to have something like the following with some modifications applied:

```java
    public static Pipeline getPipeline(final String owner) {
        return getPipeline(ID, owner, ID);
    }
```